tinydrive/tinydrive-api/standalone.md

```diff
@@ -83,6 +142,40 @@ tinydrive.start({
 });
 ```
 
+## `tinydrive.controls.start`
+
+Start is similar to pick and browse, the main difference is that users do not have a way of closing the dialog with this method. This can be useful when you want to launch {{site.cloudfilemanager}} from an url where there is no application to insert the files into. This might be useful when you want to launch {{site.cloudfilemanager}} from an url but not as part of a bigger application. A [Control Object](#controlobject) is returned which control the {{site.cloudfilemanager}} lifecycle, which it no longer does by itself. For a complete list of available settings to pass in to this method see [Picker settings](#pickersettings).
```

Comment:
```suggestion
Start is similar to pick and browse, but users do not have a way of closing the dialog with this method. This can be useful when you want to launch {{site.cloudfilemanager}} from a URL where there is no application to insert the files into. This might be useful when you want to launch {{site.cloudfilemanager}} from a URL, but not as part of a bigger application. A [Control Object](#controlobject) is returned which control the {{site.cloudfilemanager}} lifecycle, which is not handled by {{site.cloudfilemanager}} when using the `controls` APIs. For a complete list of available settings to pass in to this method, see: [Picker settings](#pickersettings).
```